invenio_rdm_records/theme/templates/invenio_rdm_records/details/list_people.html

```diff
@@ -0,0 +1,27 @@
+{#
+    Copyright (C) 2020 CERN.
+    Copyright (C) 2020 Northwestern University.
+
+    Invenio RDM Records is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% macro list_creators_like(creators_like) %}
```

Comment:
Silly question, why `creators_like`?

Reply:
Because it applies to list of creators *and* contributors (any object that conforms to the CreatorSchema interface)

Reply:
I am not sure though that the `like` suffix would be enough to remember why...I would suggest to leave a comment there maybe? @ppanero WDYT?

Reply:
Would there be a different name that is more informative?  What about:

```jinja
{% macro list_creators_or_contributors(creators_or_contributors) %}
``` 

?

Reply:
Ended up going for `list_entities` : shorter